src/components/Filterbox/Filterbox.js

```diff
@@ -1,7 +1,9 @@
 import React from "react";
+import { useNote } from "../../context/note-context";
 
 import "./filterbox.css";
 export const Filterbox = (props) => {
+  const { noteListDisptach, noteListState } = useNote();
   return (
```

Comment:
your code is also correct just a suggestion :-  you can desturcture noteListState here like const { sortbydate , sortbypriority , tags}= noteListState